tests/resources/mlflow-test-plugin/setup.py

```diff
@@ -8,7 +10,7 @@
     packages=find_packages(),
     # Require MLflow as a dependency of the plugin, so that plugin users can simply install
     # the plugin & then immediately use it with MLflow
-    install_requires=["mlflow"],
+    install_requires=["mlflow" if "MLFLOW_SKINNY" not in os.environ else "mlflow-skinny"],
```

Comment:
Is this strictly necessary? We publish this test plugin example in our docs on `mlflow.org`. The less complexity / confusion we can add, the better.

Reply:
With Harupy's new change it won't be